docsrc/user_guide/dynamo_export.rst

```diff
@@ -0,0 +1,72 @@
+.. _getting_started_with_dynamo:
```

Comment:
To be reflected in the index's `ref`, this would need to be changed to `_dynamo_export`